packages/core/src/tests/menu/menu.e2e.ts

```diff
@@ -8,51 +8,62 @@
  */
 
 import { expect } from '@playwright/test';
-import { test, viewPorts } from '@utils/test';
-
-test('should render menu items with a[href]', async ({ mount, page }) => {
-  await page.setViewportSize(viewPorts.lg);
-  await mount(`
-  <ix-basic-navigation>
-    <ix-menu>
-      <a href="#link1">
-        <ix-menu-item>Link 1</ix-menu-item>
-      </a>
-      <ix-menu-category label="Sub">
-        <a href="#link2">
-          <ix-menu-item>Link 2</ix-menu-item>
-        </a>
-        <a href="#link3">
-          <ix-menu-item active>Link 3</ix-menu-item>
-        </a>
-        <a href="#link4">
-          <ix-menu-item>Link 4</ix-menu-item>
-        </a>
-        <a href="#link5">
-          <ix-menu-item>Link 5</ix-menu-item>
-        </a>
-      </ix-menu-category>
-    </ix-menu>
-  </ix-basic-navigation>`);
-
-  const basicNavigationElement = page.locator('ix-basic-navigation');
-
-  const category = page.locator('ix-menu-category');
-  await category.click();
-
-  const link1 = page.getByText('Link 1');
-  await expect(link1).toBeVisible();
-
-  const link2 = page.getByText('Link 2');
-  await expect(link2).toBeVisible();
-
-  await link2.hover();
-
-  await page.waitForTimeout(1000);
-
-  expect(
-    await basicNavigationElement.screenshot({
-      animations: 'disabled',
-    })
-  ).toMatchSnapshot();
+import { regressionTest, viewPorts } from '@utils/test';
+
+regressionTest.describe('menu', () => {
+  regressionTest('basic', async ({ page }) => {
+    await page.setViewportSize(viewPorts.lg);
+    await page.goto('menu/basic');
+
+    const basicNavigationElement = page.locator('ix-basic-navigation');
+
+    const link1 = page.getByText('Link 1');
+    await expect(link1).toBeVisible();
+
+    const link2 = page.getByText('Link 2');
+    await expect(link2).toBeVisible();
+
+    await link2.hover();
+
+    await page.waitForTimeout(1000);
+
+    expect(
+      await basicNavigationElement.screenshot({
+        animations: 'disabled',
+      })
+    ).toMatchSnapshot();
+  });
+
+  regressionTest(
+    'category open on expand when initially closed',
+    async ({ page }) => {
+      await page.setViewportSize(viewPorts.lg);
+      await page.goto('menu/basic');
+
+      const basicNavigationElement = page.locator('ix-basic-navigation');
+
+      const categoryElement = page
+        .locator('ix-menu-item')
+        .filter({ hasText: 'Sub' });
+      await categoryElement.click();
+
+      const collapseButton = page.getByRole('button', {
+        name: 'Double Chevron Right',
```

Comment:
Once #1112 is merged we can change this to:
```suggestion
        name: 'Double Chevron Left',
```